src/azul/service/manifest_service.py

```diff
@@ -704,15 +704,15 @@ def write_to(self, output: IO[str]) -> Optional[str]:
             url = furl(config.service_endpoint(),
                        path=f'/repository/files/{uuid}',
                        args=dict(version=version, catalog=self.catalog))
-            name = self._validate(name)
+            name = self._validate_path(name)
             output.write(f'url={self._option(url.url)}\n'
                          f'output={self._option(name)}\n\n')
         return None
 
     @classmethod
-    def _validate(cls, path: str):
+    def _validate_path(cls, path: str):
```

Comment:
Return type?

Reply:
Since the method above `_option` didn't have a returned type, I figured I also leave it out for this one. But it is more consisten with the rest of the methods if we include it, ty!